_layouts/post.html

```diff
@@ -1,58 +1,66 @@
-            <img src="{{ page.image }}" alt="post logo">
-            <label>{{ page.title }}</label>
-            <p>
-                {% assign words = content | number_of_words %}
-                {% if words < 360 %}
-                  1 min read.
-                {% else %}
-                  {{ words | divided_by:180 }} mins read.
-                {% endif %} 
-                Updated {{ page.date | date_to_string }}
-            </p>
-            {% if page.tags %}
-              {% for tag in page.tags %}
-                <span class="badge badge-pill badge-default">
-                  {% capture tag-name %}{{tag | upcase }}{% endcapture %}
-                  {% if tag-name=="AWS" or tag-name=="GCP"  or tag-name=="GKE" %} {{ tag-name }} {% else %} {{tag-name | capitalize}}{% endif %}</span>
-              {% endfor %}
-            {% endif %}
-          </div>
-          <div class="content">
-              {{ content }}
-          </div>
-
-    </div>
-    <div class="col-md-2">
-        <div class="card contact-us-card text-center">
-          <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
-            <div class="card-body">
-              <p class="card-title">Want to learn more? Feel free to contact us for support</p>
-              <a href="#" class="btn btn-primary">Support</a>
-              <a href="#">Dismiss</a>
-            </div>
-          </div>
     </div>
+    <div class="col-md-7 guides-section-white">
+      <ol class="breadcrumb text-center">
+        {% assign crumbs = page.url | remove:'/index.html' | split: '/' %}
+        <a href="/guides" ga-event-category="guide-{{ post.title | downcase}}" ga-event-action="deployment-guides">Dev Ops Guides</a> 
+        {% for crumb in crumbs offset: 2 %}
+          {% if forloop.last %}
+           <a href="#"> / {{ crumb | replace:'-',' ' | remove:'.html' | capitalize }}</a>
+          {% else %}
+            <a href="{{ crumb }}">/ {{ crumb | replace:'-',' ' | remove:'.html' | capitalize }}</a>
+          {% endif %}
+        {% endfor %}      
+      </ol>
+      <div class="post-title text-center">
+        <img class="img-center" src="{{ page.image }}" alt="post logo">
+        <label>{{ page.title }}</label>
+        <p>
+          {% assign words = content | number_of_words %}
+          <!-- https://help.medium.com/hc/en-us/articles/214991667-Read-time -->
+          {% if words < 265 %}
+            1 min read.
+          {% else %}
+            {{ words | divided_by:180 }} mins read.
+          {% endif %} 
+          Updated {{ page.date | date_to_string }}
+        </p>
+        {% if page.tags %}
+          {% for tag in page.tags %}
+            <span class="badge badge-pill badge-default">
+              {% capture tag-name %}{{tag | upcase }}{% endcapture %}
+              {% if tag-name=="AWS" or tag-name=="GCP"  or tag-name=="GKE" %} {{ tag-name }} {% else %} {{tag-name | capitalize}}{% endif %}</span>
```

Comment:
Tags should be clickable and go deployment guides list page, filtered to that tag. Can be done in separate PR. Please create Jira card.

Reply:
+1 making these clickeable in a separate PR. 

Concerning this specific code, why are there so many ifs? Is it because "AWS" etc is already upper case? One danger here is that we are not going to have full control of all the tags. What if someone adds a blog post with new tag that's also an acronym? Maybe we should expect them to be properly capitalized at the metadata of the posts? And then downcase as necessary (if we use it in urls for example)?